documentation/source/contributors.rst

```diff
@@ -6,7 +6,14 @@ Contributors
    McGregor
    Grimwood
 
-cocotb was developed by `Potential Ventures <https://potential.ventures>`_ with the support of
-`Solarflare Communications Ltd <https://www.solarflare.com/>`_
-and contributions from Gordon McGregor and Finn Grimwood
-(see `contributors <https://github.com/cocotb/cocotb/graphs/contributors>`_ for the full list of contributions).
+cocotb is developed and maintained by an active community.
+Our GitHub repository contains a list of all `contributors <https://github.com/cocotb/cocotb/graphs/contributors>`_.
+
+Since mid-2018 cocotb is an independent project under the umbrella of the
```

Comment:
```suggestion
Since mid-2018, cocotb is an independent project under the umbrella of the
```
maybe?

Reply:
Sorry, I missed this one. Should be fine either way.